docs/contributing/RoslynDebugging.md

```diff
@@ -0,0 +1,16 @@
+# Debugging with experimental Roslyn bits
+Sometimes it may be necessary to make changes in [`dotnet/roslyn`](https://github.com/dotnet/roslyn), and react to the changes in this repo. The following are steps which outline the general process in using Roslyn development `dll`s with Razor Tooling.
+
+## Steps:
+1. Checkout [`dotnet/roslyn`](https://github.com/dotnet/roslyn)
+2. `./Restore.cmd`
+3. Make the desired changes in `dotnet/roslyn`
+4. `./Build.cmd -pack` => `-pack` instructs for the creation of the NuGet packages
+5. You should see the generated packages in the `roslyn\artifacts\packages\Debug\Release` directory. Take note of the package versions (ie. `Microsoft.CodeAnalysis.Workspaces.Common.3.8.0.nupkg` => `3.8.0`)
+6. Open the `aspnetcore-tooling/NuGet.config` and add the local package source `<add key="Roslyn Local Package source" value="[PATH_TO_ROSLYN_REPO]\artifacts\packages\Debug\Release" />`
+7. Open `aspnetcore-tooling/eng/Versions.props` and update all the `Tooling_*` versions to the version noted in step 5, except for  `Tooling_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion`.
+
+## Notes:
+- Running the `dotnet/roslyn` project uses the `RoslynDev` root suffix Visual Studio Hive.
```

Comment:
I think people not deep into our repo will find this statement confusing, though I have no specific suggestions for improving it.

Reply:
This is how roslyn handles it:
> If you're familiar with the idea of Visual Studio hives, we deploy into the RoslynDev root suffix.
https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building,%20Debugging,%20and%20Testing%20on%20Windows.md

I'll update to reflect something similar